fedimint-cli/src/client.rs

```diff
@@ -99,6 +100,7 @@ pub enum ClientCmd {
         oob_notes: Vec<OOBNotes>,
     },
     /// Create a lightning invoice to receive payment via gateway
+    #[clap(hide = true)]
```

Comment:
Clever, so it keeps working in tests while we teach people to use the new CLI commands …

Reply:
We should still try to, over time, migrate tests so we actually test the real thing.